iocore/net/P_SSLNetVConnection.h

```diff
@@ -295,7 +302,7 @@ class SSLNetVConnection : public UnixNetVConnection
     HANDSHAKE_HOOKS_DONE
   } sslHandshakeHookState;
 
-  const SSLNextProtocolSet *npnSet;
+  mutable SSLNextProtocolSet *npnSet;
```

Comment:
It does not. My initial implementation required this, then I forgot to change this back